sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipTrunk.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.PhoneNumbers.SipRouting
+{
+    /// <summary> Gets or sets SIP signaling port of the trunk. </summary>
+    [CodeGenClient("Trunk")]
+    public partial class SipTrunk
+    {
+        /// <summary> Initializes a new instance of SipTrunk. </summary>
+        /// <param name="fqdn">Name of the trunk</param>
+        /// <param name="sipSignalingPort"> Gets or sets SIP signaling port for the gateway. </param>
```

Comment:
`/// <param name="sipSignalingPort"> Gets or sets [...]` I don't see how this parameter can actually get the port number.

Reply:
Fixed comments.